vassal-app/src/main/java/VASSAL/configure/AutoConfigurer.java

```diff
@@ -152,7 +149,7 @@ else if (TranslatableStringEnum.class.isAssignableFrom(type)) {
       if (se != null) {
         final String[] validValues = se.getValidValues(target);
         final String[] i18nKeys = se.getI18nKeys(target);
-        config = new TranslatingStringEnumConfigurer(key, prompt, validValues, i18nKeys);
+        config = new TranslatingStringEnumConfigurer(key, prompt, validValues, i18nKeys, se.isDisplayNames());
```

Comment:
Shouldn't need to be changed, since se.isDisplayName() will always be false anyway. Change the old Constructors on TranslatingStringEnumConfigurer to set the displayNames field to false.